backend/src/auth/auth.service.ts

```diff
@@ -19,7 +20,8 @@ export class AuthService {
     async login(user: User) {
         const payload = { username: user.username, sub: user.uuid };
         return {
-            acceess_token: this.jwtService.sign(payload)
+            user: user as ResponseUserDto,
```

Comment:
`as` is not real casting variable to another type, it won't strip any data, we should use `new ResponseUserDto(user)` here.